activity/helper.go

```diff
@@ -0,0 +1,56 @@
+package activity
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/puppetlabs/wash/datastore"
+	"github.com/shirou/gopsutil/process"
+	log "github.com/sirupsen/logrus"
+)
+
+// Cache pid to process names. This may end up getting the wrong process name if there are
+// lots of new processes being created constantly, but makes fast things a *lot* faster.
+var pidToExec = datastore.NewMemCache()
+
+// JournalForPID converts a process ID to a journal ID, and the command invocation of that
```

Comment:
This comment should probably be re-worded. Maybe something like "JournalForPID creates a journal that can be used to record all Wash-related activity induced by the given process ID. The journal ID is formatted as `<pid>-<name>-<createtime>`" (I like explicitly specifying the format better because it's more concise)